workflow/github.md

```diff
@@ -1,2 +1,138 @@
 # Working on GitHub
 
+This document outlines the rules to contribute and to create issues to a project on the organisation [Canonical-websites](https://github.com/canonical-websites/)
+
+## Table of content
+
+* [How can I contribute?](#how-can-i-contribute-)
+  + [Report Bugs](#report-bugs)
+  + [Suggest Enhancements](#suggest-enhancements)
+  + [Work on an issue](#work-on-an-issue)
+* [Pull Requests](#pull-requests)
+  + [First contribution](#first-contribution)
+  + [Styleguides](#styleguides)
+    - [Commits messages](#commits-messages)
+    - [HTML](#html)
+    - [JS](#js)
+    - [Stylesheets](#stylesheets)
+    - [Python](#python)
+    - [Documentation](#documentation)
+  + [Merging process](#merging-process)
+* [Templates](#templates)
+  + [Issues template](#issues-template)
+  + [Pull-request template](#pull-request-template)
+
+## How can I contribute?
+
+In the Canonical web-sites organisation we work with Github. All types of work are tracked as GitHub issues. You will find in this section how you can contribute to this organisation.
```

Comment:
"the Canonical web-sites organisation" is not quite right - this repo should lay out practices for `canonical-webteam`, `canonical-websites`, `ubuntudesign` and *maybe* `canonical-docs`. I suppose technically we're the "Canonical design team", but that might confuse people. In other places I think I've simply says "The web-team", but that's a very loosely defined term, and might be confused with [the web squad](https://github.com/ubuntudesign/web-squad).

I need help. What shall we use as the loose umbrella term for all the people who might follow these practices?

Reply:
I don't mind the term `webteam`. As this org is called `canonical-webteam` already.